src/olympia/devhub/templates/devhub/new-landing/index.html

```diff
@@ -37,20 +37,12 @@
         data-raven-urls="{{ settings.RAVEN_ALLOW_LIST }}"
         {% block bodyattrs %}{% endblock %}>
 
+    {% set user_authenticated = not request.user.is_authenticated() %}
+
     {% include 'devhub/new-landing/components/navigation.html' %}
 
-    {% if not request.user.is_authenticated() %}
-      <section class="DevHub-Header">
-        <h1>{{ _('Customize Your Firefox') }}</h1>
-        <p>
-          {% trans %}
-          Add-ons let millions of Firefox users enhance their browsing experience.
-          If you know <a href="#">HTML</a>, <a href="#">JavaScript</a>, and <a href="#">CSS</a>,
-          you already have all the necessary skills to make a great add-on.
-          {% endtrans %}
-        </p>
-        <a href="#" class="Button Button--primary">Learn How to make an Add-on</a>
-      </section>
+    {% if not user_authenticated %}
```

Comment:
This should be `{% if user_authenticated %}` I think, because `user_authenticated === not request.user.is_authenticated()` and previously this check for `not request.user.is_authenticated()`... now it's checking for `not not request.user.is_authenticated()`.

Maybe change the variable name to `not_authenticated` or `anonymous_user`?